--- rpki-client-8.5.old/configure	2023-07-29 22:54:42.000000000 +0200
+++ rpki-client-8.5/configure	2023-08-16 23:07:13.744941298 +0200
@@ -13821,6 +13821,8 @@
 		$as_echo "#define HAVE_LANDLOCK 1" >>confdefs.h
 
 	fi
+
+fi
 	 if test "x$ac_cv_header_linux_landlock_h" = xyes; then
   HAVE_LANDLOCK_TRUE=
   HAVE_LANDLOCK_FALSE='#'
@@ -13829,8 +13831,6 @@
   HAVE_LANDLOCK_FALSE=
 fi
 
-fi
-
 
 # Check whether --with-user was given.
 if test "${with_user+set}" = set; then :
